docs/content/modeling/getting-started.mdx

```diff
@@ -83,17 +72,13 @@ When you are modeling, you need to answer a more general question:
   />
 </div>
 
-If you can answer that question for all types of objects in your system, then you can codify that into an authorization model.
-
-Let's get started!
+Answering that question for all object types in your system codifies your authorization model.
 
 ---
 
 ## A Process For Defining Authorization Models
 
-Defining an authorization model requires codifying an answer to the question "why could user U perform an action A on an object O?" for all use cases or actions in your system. This is an iterative process. For the purpose of this guide, we'll go through one iteration of this process using a simplified Google Drive like system as an example.
-
-Steps for defining your authorization model:
+Below is one iteration of the codification process, using a simplified Google Drive-like system as an example. The following steps define your authorization model:
```

Comment:
maybe codification is the wrong term here, can we replace it with something else

```suggestion
Below is one iteration of defining an authorization model for your system, using a simplified Google Drive-like system as an example. The following are the steps for this iteration:
```